journal/updated/01-journal.adoc

```diff
@@ -767,251 +767,270 @@ nevertheless, afterward it yields the peaceable fruit of righteousness,
 
 Then the priest left me, and the justices asked me, seeing I would not take the oaths,
-whether I would give bail; and said,
+whether I would give bail; and said
 they would take my father`'s bail for my good behaviour.
-I told them, my cause was just, and I was innocent, and would give them no bail,
-for truth binds me to my good behaviour.
-Then the high-sheriff, a very fair man, told me, I was a strange man,
+I told them, "`My cause is just, and I am innocent, and will give you no bail,
+for truth binds me to my good behaviour.`"
+Then the high-sheriff, a very fair man, told me I was a strange man,
 and of a strange persuasion, to come with my hat upon my head among them,
-and would not take the oaths nor give bail.
-You know, said he, that Paul said to Festus, Noble Festus.
-I told him, that Paul had tried Festus, but I had not as yet tried him; and it might be,
-that I might speak of him, Noble Sheriff, upon this they were most of them very pleasant.
-He asked me, whom I did know there; there were several justices my relations present,
-who very well knew me, but I made mention of none of them; but told them,
+and not take the oaths nor give bail.
+"`You know,`" said he, "`that Paul said to Festus, '`Noble Festus.`'`"
+I told him that Paul had tried Festus, but I had not yet tried him;
+and it may be, that after I had tried him, I might say to him "`Noble Sheriff;`"
+and upon saying this most of them seemed very pleasant.
+He asked me whom I knew there; there were several justices present
+who were my relations, and who knew me very well,
+but I made mention of none of them, but told them
 I knew the chief magistrate of the town, Charles Jones, whose prisoner I was.
-Then they called for him, and asked him, whether he knew me; he said he did, very well,
-for I was born and bred in the town among them,
-and was a very honest young man all along; but, said he, what devil he has now,
-I know not.
-They asked him, whether I was his prisoner; he said I was.
-Then, said the justices, take him again to your custody.
-
-As I was going out of the room, I told them, that I brought a good hat on my head there,
-but was going away without it, for some of the baser sort had conveyed it away,
-but the justices made diligent search about it; so it was brought me again,
-and put upon my head, and they parted with me very friendly,
-and the town magistrate took me a little from them,
+Then they called for him, and asked him whether or not he knew me;
+he said he did, very well, for I was born and bred in the town among them,
+and was a very honest young man in all my ways. "`But,`" said he,
+"`what devil he has now, I know not.`"
+They asked him whether I was his prisoner or not; he said I was.
+"`Then,`" said the justices, "`take him again into your custody.`"
+
+As I was going out of the room, I told them
+that I had come there wearing a good hat on my head,
+but was going away without it, for some people of the base sort
```

Comment:
maybe: for some people of the **baser** sort had **taken** it away. But the justices made diligent search **for** it, so that it was